_includes/navbar.html

```diff
@@ -1,14 +1,17 @@
 <header class="site-header">
     
-	<div class="branding">
-		<a href="{{ site.baseurl }}/">
-			<img class="avatar" src="{{ site.baseurl }}/{{ site.theme_settings.avatar }}" alt=""/>
-		</a>
+    <div style="padding: 5px; float: left;">
+        
+        <div class="branding" style="display: inline; float: left;">
```

Comment:
Can you move inline style like this into `assets/css/main.scss`?

Reply:
yep sure thing.

Reply:
figured the changes would be more visible here, but they should definately live in the file i just forgot.

Reply:
where should i put it? im thinking in navbar but also looking through the other files.

Reply:
maybe assets/css/main?

Reply:
@alexanderankin right, like I mentioned in my first comment :grin: